docs_outdoornav_user_manual/web_user_interface/ui_map_mode.mdx

```diff
@@ -0,0 +1,363 @@
+---
+title: Web UI Map Mode
+sidebar_label: Web UI Map Mode
+sidebar_position: 3
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
+![](/img/outdoornav_images/gps_danger.png)
+
+Ensure that the [Safety](../safety.mdx) document has been
+read and the Operator is aware of possible hazards when using this product as well as the safety
+methods that can be used to stop the moving UGV.
+
+The Map Mode of OutdoorNav Software uses a set of robotic
+navigation modules that allows the Operator(s) to define drivable paths and 
```

Comment:
"drivable" or "driveable"? I'm ok either way, but I think we use different spellings in different files. Can you do a search-and-replace across all files to ensure consistency for one or the other?

Reply:
Switched to driveable and did a search-and-replace for it.